_posts/2019-12-09-ebcdic-character-format-guide.markdown

```diff
@@ -31,52 +31,35 @@ This post is mainly going to deal with fixed width EBCDIC records.
 
 At it's heart EBCDIC is just a character encoding, in simple terms that is how we map a number onto character as computers only store numbers i.e. 61:a, 62:b, 63:c.
```

Comment:
its rather than it's.
Perhaps liken EBCDIC to ASCII at this point: At its heart, both EBCDIC (Extended Binary Coded Decimal Interchange Code) and ASCII (American Standard Code for Information Interchange) are methods of character encoding. In simple terms, they are a way of translating a binary number sequence into letters and numbers: ie 61 = a, 62 = b, 63 = c.